File: app/src/main/java/modernjava/ex02.java
package modernjava;

import java.util.Arrays;
import java.util.List;

public class ex02 {
    public static void main(String[] args) {
        List<String> list = Arrays.asList("Apple", "Banana", "Cherry");
        list.sort((a, b) -> b.compareTo(a));
        list.forEach(System.out::println);
    }
}

File: app-kotlin/src/main/kotlin/modernjava/ex02.kt
package modernjava

class ex02 {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val list = mutableListOf("Apple", "Banana", "Cherry")
            list.sortByDescending { it }
            list.forEach { println(it) }
        }
    }
}
